contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -367,6 +400,8 @@ contract NodeOperatorsRegistry is AragonApp, Versioned {
         _saveOperatorSigningKeysStats(_nodeOperatorId, signingKeysStats);
 
         emit VettedSigningKeysCountChanged(_nodeOperatorId, vettedSigningKeysCountAfter);
+
+        _updateTotalMaxValidatorsCount(_nodeOperatorId);
         _increaseValidatorsKeysNonce();
```

Comment:
Super minor, I think, but according to our rules, nonce change might not be required. When target limits & penalties applied to node operator is less than vetted validators count